CHANGELOG.md

```diff
@@ -4,10 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.14.1]
+### Added
+- add 2 icons for moving to top or bottom
```

Comment:
is this message correct? it said add 2 icon but it actually removing one icon on this PR

Reply:
yeah, i think i need to change the message